composeApp/src/commonMain/kotlin/band/effective/office/elevator/ui/authorization/authorization_profile/AuthorizationProfileScreen.kt

```diff
@@ -0,0 +1,320 @@
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Lock
+import androidx.compose.material.icons.filled.Person
+import androidx.compose.material.icons.outlined.Close
+import androidx.compose.material.icons.rounded.ArrowBack
+import androidx.compose.material3.Divider
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.focus.onFocusChanged
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import band.effective.office.elevator.MainRes
+import band.effective.office.elevator.components.OutlinedTextColorsSetup
+import band.effective.office.elevator.components.PrimaryButton
+import band.effective.office.elevator.getDefaultFont
+import band.effective.office.elevator.textGrayColor
+import band.effective.office.elevator.theme_light_primary_stroke
+import band.effective.office.elevator.ui.authorization.authorization_phone.store.AuthorizationPhoneStore
+import band.effective.office.elevator.ui.authorization.authorization_profile.store.AuthorizationProfileStore
+import band.effective.office.elevator.ui.authorization.components.AuthSubTitle
+import band.effective.office.elevator.ui.authorization.components.AuthTabRow
+import band.effective.office.elevator.ui.authorization.components.AuthTitle
+import dev.icerock.moko.resources.compose.painterResource
+import dev.icerock.moko.resources.compose.stringResource
+
+@Composable
+fun AuthorizationProfileScreen(component: AuthorizationProfileComponent) {
+    LaunchedEffect(component) {
+        component.label.collect { label ->
+            when (label) {
+                AuthorizationProfileStore.Label.AuthorizationProfileFailure -> TODO()
+                AuthorizationProfileStore.Label.AuthorizationProfileSuccess -> TODO()
+                AuthorizationProfileStore.Label.OpenTGAuthorization -> component.onOutput(
+                    AuthorizationProfileComponent.Output.OpenTGScreen
+                )
+
+                AuthorizationProfileStore.Label.ReturnInPhoneAuthorization -> component.onOutput(
+                    AuthorizationProfileComponent.Output.OpenPhoneScreen
+                )
+            }
+        }
+    }
+
+    AuthorizationProfileComponent(onEvent = component::onEvent)
+}
+
+@Composable
+fun AuthorizationProfileComponent(onEvent: (AuthorizationProfileStore.Intent) -> Unit) {
+//    region::Variables
+    val isError1 = remember { mutableStateOf(false) }
+    val isError2 = remember { mutableStateOf(false) }
+    val name = remember { mutableStateOf("") }
```

Comment:
these variables should be in your state. And change from via the `onEvent()` method